[debug] [2025-08-13T02:10:41.487Z] [Firebase Plugin] Activating Firebase extension.
[info] [Firebase Plugin] Checked firebase-tools, is up to date!
[debug] [2025-08-13T02:10:44.722Z] [Firebase Plugin] requireAuthWrapper
[debug] [2025-08-13T02:10:44.723Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2025-08-13T02:10:44.724Z] > authorizing via signed-in user (smartdisputecanada@gmail.com)
[debug] [2025-08-13T02:10:44.730Z] No OAuth tokens found
[debug] [2025-08-13T02:10:44.731Z] No OAuth tokens found
[debug] [2025-08-13T02:10:44.731Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2025-08-13T02:10:44.732Z] > authorizing via signed-in user (smartdisputecanada@gmail.com)
[debug] [2025-08-13T02:10:44.733Z] [Firebase Plugin] User found:  smartdisputecanada@gmail.com
[info] [Firebase Plugin] (Core:Project) New user detected, fetching projects
[debug] [2025-08-13T02:10:44.735Z] No OAuth tokens found
[debug] [2025-08-13T02:10:44.737Z] >>> [apiv2][query] GET https://firebase.googleapis.com/v1beta1/projects pageSize=1000
[debug] [2025-08-13T02:10:44.796Z] [Firebase Plugin] VSCode notification server listening on port 40001
[debug] [2025-08-13T02:10:45.178Z] <<< [apiv2][status] GET https://firebase.googleapis.com/v1beta1/projects 200
[debug] [2025-08-13T02:10:45.178Z] <<< [apiv2][body] GET https://firebase.googleapis.com/v1beta1/projects [omitted]
[info] [Firebase Plugin] Starting Data Connect toolkit (version 2.10.1) on port 50001
[info] i  dataconnect: downloading dataconnect-emulator-2.10.1... {"metadata":{"emulator":{"name":"dataconnect"},"message":"downloading dataconnect-emulator-2.10.1..."}}
[debug] [2025-08-13T02:10:52.796Z] >>> [apiv2][query] GET https://storage.googleapis.com/firemat-preview-drop/emulator/dataconnect-emulator-linux-v2.10.1 
[debug] [2025-08-13T02:10:53.010Z] <<< [apiv2][status] GET https://storage.googleapis.com/firemat-preview-drop/emulator/dataconnect-emulator-linux-v2.10.1 200
[debug] [2025-08-13T02:10:53.013Z] <<< [apiv2][body] GET https://storage.googleapis.com/firemat-preview-drop/emulator/dataconnect-emulator-linux-v2.10.1 [stream]
[debug] [2025-08-13T02:10:57.328Z] 'fdc build' failed with error: Unable to build your Data Connect schema and connectors (exit code 1): E0813 02:10:57.325678   35980 cmd.go:43] Error loading data connect config: could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"'fdc build' failed with error: Unable to build your Data Connect schema and connectors (exit code 1): E0813 02:10:57.325678   35980 cmd.go:43] Error loading data connect config: could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-08-13T02:10:57.330Z] Ignoring unsupported arg: auto_download {"metadata":{"emulator":{"name":"dataconnect"},"message":"Ignoring unsupported arg: auto_download"}}
[debug] [2025-08-13T02:10:57.331Z] Ignoring unsupported arg: port {"metadata":{"emulator":{"name":"dataconnect"},"message":"Ignoring unsupported arg: port"}}
[debug] [2025-08-13T02:10:57.331Z] Starting Data Connect Emulator with command {"binary":"/home/user/.cache/firebase/emulators/dataconnect-emulator-2.10.1","args":["--logtostderr","-v=2","dev","--listen=localhost:50001","--config_dir=/home/user/studio/dataconnect","--enable_output_schema_extensions=true","--enable_output_generated_sdk=true"],"optionalArgs":["listen","config_dir","enable_output_schema_extensions","enable_output_generated_sdk"],"joinArgs":true,"shell":false,"port":9399} {"metadata":{"emulator":{"name":"dataconnect"},"message":"Starting Data Connect Emulator with command {\"binary\":\"/home/user/.cache/firebase/emulators/dataconnect-emulator-2.10.1\",\"args\":[\"--logtostderr\",\"-v=2\",\"dev\",\"--listen=localhost:50001\",\"--config_dir=/home/user/studio/dataconnect\",\"--enable_output_schema_extensions=true\",\"--enable_output_generated_sdk=true\"],\"optionalArgs\":[\"listen\",\"config_dir\",\"enable_output_schema_extensions\",\"enable_output_generated_sdk\"],\"joinArgs\":true,\"shell\":false,\"port\":9399}"}}
[info] i  dataconnect: Data Connect Emulator logging to dataconnect-debug.log {"metadata":{"emulator":{"name":"dataconnect"},"message":"Data Connect Emulator logging to dataconnect-debug.log"}}
[debug] [2025-08-13T02:10:57.374Z] E0813 02:10:57.373877   35985 load.go:23] Could not load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
E0813 02:10:57.374213   35985 watch.go:76] watchAllDir unable to walk dir `/home/user/studio/dataconnect`: on path "/home/user/studio/dataconnect" (DirEntry: <nil>): lstat /home/user/studio/dataconnect: no such file or directory
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0813 02:10:57.373877   35985 load.go:23] Could not load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\nE0813 02:10:57.374213   35985 watch.go:76] watchAllDir unable to walk dir `/home/user/studio/dataconnect`: on path \"/home/user/studio/dataconnect\" (DirEntry: <nil>): lstat /home/user/studio/dataconnect: no such file or directory\n"}}
[debug] [2025-08-13T02:10:57.396Z] I0813 02:10:57.396586   35985 dev.go:95] Listening on address (HTTP + gRPC): 127.0.0.1:50001
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0813 02:10:57.396586   35985 dev.go:95] Listening on address (HTTP + gRPC): 127.0.0.1:50001\n"}}
[error] [Firebase Plugin] error:  {}
[debug] [2025-08-13T02:11:07.374Z] E0813 02:11:07.374500   35985 load.go:23] Could not load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0813 02:11:07.374500   35985 load.go:23] Could not load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-08-13T02:11:17.384Z] E0813 02:11:17.383775   35985 load.go:23] Could not load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0813 02:11:17.383775   35985 load.go:23] Could not load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-08-13T02:11:27.649Z] E0813 02:11:27.374430   35985 load.go:23] Could not load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0813 02:11:27.374430   35985 load.go:23] Could not load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-08-13T02:11:37.374Z] E0813 02:11:37.374590   35985 load.go:23] Could not load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0813 02:11:37.374590   35985 load.go:23] Could not load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-08-13T02:11:47.381Z] E0813 02:11:47.381233   35985 load.go:23] Could not load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0813 02:11:47.381233   35985 load.go:23] Could not load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-08-13T02:11:57.375Z] E0813 02:11:57.375446   35985 load.go:23] Could not load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0813 02:11:57.375446   35985 load.go:23] Could not load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-08-13T02:12:07.375Z] E0813 02:12:07.375507   35985 load.go:23] Could not load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0813 02:12:07.375507   35985 load.go:23] Could not load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-08-13T02:12:17.377Z] E0813 02:12:17.377309   35985 load.go:23] Could not load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0813 02:12:17.377309   35985 load.go:23] Could not load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-08-13T02:12:27.375Z] E0813 02:12:27.375358   35985 load.go:23] Could not load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0813 02:12:27.375358   35985 load.go:23] Could not load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-08-13T02:12:37.376Z] E0813 02:12:37.375998   35985 load.go:23] Could not load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0813 02:12:37.375998   35985 load.go:23] Could not load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-08-13T02:12:47.387Z] E0813 02:12:47.384139   35985 load.go:23] Could not load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0813 02:12:47.384139   35985 load.go:23] Could not load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-08-13T02:12:57.375Z] E0813 02:12:57.374801   35985 load.go:23] Could not load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
E0813 02:12:57.374872   35985 watch.go:76] watchAllDir unable to walk dir `/home/user/studio/dataconnect`: on path "/home/user/studio/dataconnect" (DirEntry: <nil>): lstat /home/user/studio/dataconnect: no such file or directory
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0813 02:12:57.374801   35985 load.go:23] Could not load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\nE0813 02:12:57.374872   35985 watch.go:76] watchAllDir unable to walk dir `/home/user/studio/dataconnect`: on path \"/home/user/studio/dataconnect\" (DirEntry: <nil>): lstat /home/user/studio/dataconnect: no such file or directory\n"}}
[debug] [2025-08-13T02:13:07.375Z] E0813 02:13:07.375205   35985 load.go:23] Could not load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0813 02:13:07.375205   35985 load.go:23] Could not load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-08-13T02:13:17.374Z] E0813 02:13:17.374513   35985 load.go:23] Could not load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0813 02:13:17.374513   35985 load.go:23] Could not load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-08-13T02:13:27.375Z] E0813 02:13:27.374841   35985 load.go:23] Could not load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0813 02:13:27.374841   35985 load.go:23] Could not load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-08-13T02:13:37.376Z] E0813 02:13:37.376174   35985 load.go:23] Could not load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0813 02:13:37.376174   35985 load.go:23] Could not load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-08-13T02:13:39.456Z] [Firebase Plugin] Value of process.env.MONOSPACE_ENV: true
[debug] [2025-08-13T02:13:39.458Z] [Firebase Plugin] requireAuthWrapper
[debug] [2025-08-13T02:13:39.460Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2025-08-13T02:13:39.461Z] > authorizing via signed-in user (smartdisputecanada@gmail.com)
[debug] [2025-08-13T02:13:39.462Z] No OAuth tokens found
[debug] [2025-08-13T02:13:39.463Z] [Firebase Plugin] User found:  smartdisputecanada@gmail.com
[info] [Firebase Plugin] (Core:Project) New user detected, fetching projects
[debug] [2025-08-13T02:13:39.464Z] No OAuth tokens found
[debug] [2025-08-13T02:13:39.467Z] >>> [apiv2][query] GET https://firebase.googleapis.com/v1beta1/projects pageSize=1000
[debug] [2025-08-13T02:13:39.516Z] [Firebase Plugin] Value of process.env.MONOSPACE_ENV: true
[debug] [2025-08-13T02:13:39.516Z] [Firebase Plugin] requireAuthWrapper
[debug] [2025-08-13T02:13:39.517Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2025-08-13T02:13:39.517Z] > authorizing via signed-in user (smartdisputecanada@gmail.com)
[debug] [2025-08-13T02:13:39.520Z] No OAuth tokens found
[debug] [2025-08-13T02:13:39.520Z] [Firebase Plugin] User found:  smartdisputecanada@gmail.com
[info] [Firebase Plugin] (Core:Project) New user detected, fetching projects
[debug] [2025-08-13T02:13:39.522Z] No OAuth tokens found
[debug] [2025-08-13T02:13:39.522Z] >>> [apiv2][query] GET https://firebase.googleapis.com/v1beta1/projects pageSize=1000
[debug] [2025-08-13T02:13:39.882Z] <<< [apiv2][status] GET https://firebase.googleapis.com/v1beta1/projects 200
[debug] [2025-08-13T02:13:39.884Z] <<< [apiv2][body] GET https://firebase.googleapis.com/v1beta1/projects [omitted]
[debug] [2025-08-13T02:13:39.886Z] <<< [apiv2][status] GET https://firebase.googleapis.com/v1beta1/projects 200
[debug] [2025-08-13T02:13:39.887Z] <<< [apiv2][body] GET https://firebase.googleapis.com/v1beta1/projects [omitted]
[debug] [2025-08-13T02:13:47.376Z] E0813 02:13:47.376076   35985 load.go:23] Could not load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0813 02:13:47.376076   35985 load.go:23] Could not load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-08-13T02:13:57.375Z] E0813 02:13:57.374984   35985 load.go:23] Could not load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0813 02:13:57.374984   35985 load.go:23] Could not load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-08-13T02:14:07.376Z] E0813 02:14:07.376002   35985 load.go:23] Could not load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0813 02:14:07.376002   35985 load.go:23] Could not load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-08-13T02:14:17.374Z] E0813 02:14:17.374524   35985 load.go:23] Could not load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0813 02:14:17.374524   35985 load.go:23] Could not load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-08-13T02:14:27.374Z] E0813 02:14:27.374622   35985 load.go:23] Could not load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0813 02:14:27.374622   35985 load.go:23] Could not load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-08-13T02:14:37.378Z] E0813 02:14:37.378192   35985 load.go:23] Could not load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0813 02:14:37.378192   35985 load.go:23] Could not load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-08-13T02:14:47.376Z] E0813 02:14:47.376105   35985 load.go:23] Could not load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0813 02:14:47.376105   35985 load.go:23] Could not load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-08-13T02:14:57.375Z] E0813 02:14:57.374723   35985 load.go:23] Could not load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
E0813 02:14:57.374788   35985 watch.go:76] watchAllDir unable to walk dir `/home/user/studio/dataconnect`: on path "/home/user/studio/dataconnect" (DirEntry: <nil>): lstat /home/user/studio/dataconnect: no such file or directory
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0813 02:14:57.374723   35985 load.go:23] Could not load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\nE0813 02:14:57.374788   35985 watch.go:76] watchAllDir unable to walk dir `/home/user/studio/dataconnect`: on path \"/home/user/studio/dataconnect\" (DirEntry: <nil>): lstat /home/user/studio/dataconnect: no such file or directory\n"}}
[debug] [2025-08-13T02:15:07.382Z] E0813 02:15:07.381550   35985 load.go:23] Could not load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0813 02:15:07.381550   35985 load.go:23] Could not load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-08-13T02:15:17.375Z] E0813 02:15:17.374964   35985 load.go:23] Could not load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0813 02:15:17.374964   35985 load.go:23] Could not load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-08-13T02:15:27.375Z] E0813 02:15:27.375155   35985 load.go:23] Could not load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0813 02:15:27.375155   35985 load.go:23] Could not load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-08-13T02:15:37.376Z] E0813 02:15:37.376148   35985 load.go:23] Could not load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0813 02:15:37.376148   35985 load.go:23] Could not load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-08-13T02:15:47.376Z] E0813 02:15:47.376095   35985 load.go:23] Could not load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0813 02:15:47.376095   35985 load.go:23] Could not load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-08-13T02:15:57.375Z] E0813 02:15:57.375290   35985 load.go:23] Could not load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0813 02:15:57.375290   35985 load.go:23] Could not load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-08-13T02:16:07.383Z] E0813 02:16:07.383160   35985 load.go:23] Could not load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0813 02:16:07.383160   35985 load.go:23] Could not load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-08-13T02:16:17.376Z] E0813 02:16:17.376165   35985 load.go:23] Could not load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0813 02:16:17.376165   35985 load.go:23] Could not load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-08-13T02:16:27.375Z] E0813 02:16:27.375094   35985 load.go:23] Could not load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0813 02:16:27.375094   35985 load.go:23] Could not load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-08-13T02:16:37.374Z] E0813 02:16:37.374529   35985 load.go:23] Could not load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0813 02:16:37.374529   35985 load.go:23] Could not load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-08-13T02:16:47.374Z] E0813 02:16:47.374542   35985 load.go:23] Could not load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0813 02:16:47.374542   35985 load.go:23] Could not load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-08-13T02:16:57.375Z] E0813 02:16:57.375439   35985 load.go:23] Could not load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
E0813 02:16:57.375508   35985 watch.go:76] watchAllDir unable to walk dir `/home/user/studio/dataconnect`: on path "/home/user/studio/dataconnect" (DirEntry: <nil>): lstat /home/user/studio/dataconnect: no such file or directory
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0813 02:16:57.375439   35985 load.go:23] Could not load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\nE0813 02:16:57.375508   35985 watch.go:76] watchAllDir unable to walk dir `/home/user/studio/dataconnect`: on path \"/home/user/studio/dataconnect\" (DirEntry: <nil>): lstat /home/user/studio/dataconnect: no such file or directory\n"}}
[debug] [2025-08-13T02:17:07.383Z] E0813 02:17:07.383191   35985 load.go:23] Could not load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0813 02:17:07.383191   35985 load.go:23] Could not load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-08-13T02:17:17.376Z] E0813 02:17:17.376174   35985 load.go:23] Could not load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0813 02:17:17.376174   35985 load.go:23] Could not load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-08-13T02:17:27.374Z] E0813 02:17:27.374672   35985 load.go:23] Could not load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0813 02:17:27.374672   35985 load.go:23] Could not load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-08-13T02:17:37.383Z] E0813 02:17:37.383650   35985 load.go:23] Could not load config from "/home/user/studio/dataconnect": could not find dataconnect.yaml in `/home/user/studio/dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0813 02:17:37.383650   35985 load.go:23] Could not load config from \"/home/user/studio/dataconnect\": could not find dataconnect.yaml in `/home/user/studio/dataconnect`\n"}}
[debug] [2025-08-13T02:17:47.376Z] I0813 02:17:47.376491   35985 load.go:37] Reloading schema and connectors...
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0813 02:17:47.376491   35985 load.go:37] Reloading schema and connectors...\n"}}
[debug] [2025-08-13T02:17:47.450Z] I0813 02:17:47.450090   35985 control.go:84] [/emulator/studio 3c0e] UpdateResources(): done
Schema: sources: schema/schema.gql [1049B] 
Connector "default": sources: 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0813 02:17:47.450090   35985 control.go:84] [/emulator/studio 3c0e] UpdateResources(): done\nSchema: sources: schema/schema.gql [1049B] \nConnector \"default\": sources: \n"}}
[debug] [2025-08-13T02:17:47.534Z] I0813 02:17:47.534229   35985 collector.go:107] schema extensions wrote into "/home/user/studio/dataconnect/.dataconnect/schema"
Generated sources: main/query.gql [7747B] main/mutation.gql [21010B] prelude.gql [73172B] main/implicit.gql [3592B] main/input.gql [54370B] main/relation.gql [30196B] 
I0813 02:17:47.534305   35985 load.go:115] Finished reloading
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0813 02:17:47.534229   35985 collector.go:107] schema extensions wrote into \"/home/user/studio/dataconnect/.dataconnect/schema\"\nGenerated sources: main/query.gql [7747B] main/mutation.gql [21010B] prelude.gql [73172B] main/implicit.gql [3592B] main/input.gql [54370B] main/relation.gql [30196B] \nI0813 02:17:47.534305   35985 load.go:115] Finished reloading\n"}}
[debug] [2025-08-13T02:18:07.375Z] I0813 02:18:07.375138   35985 load.go:37] Reloading schema and connectors...
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0813 02:18:07.375138   35985 load.go:37] Reloading schema and connectors...\n"}}
[debug] [2025-08-13T02:18:07.421Z] E0813 02:18:07.421366   35985 load.go:101] Errors generating SDK: [connector "default"]: invalid Data Connect Connector no query found in connector
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0813 02:18:07.421366   35985 load.go:101] Errors generating SDK: [connector \"default\"]: invalid Data Connect Connector no query found in connector\n"}}
[debug] [2025-08-13T02:18:07.422Z] I0813 02:18:07.422631   35985 collector.go:107] schema extensions wrote into "/home/user/studio/dataconnect/.dataconnect/schema"
Generated sources: main/input.gql [54370B] main/relation.gql [30196B] main/query.gql [7747B] main/mutation.gql [21010B] prelude.gql [73172B] main/implicit.gql [3592B] 
I0813 02:18:07.422667   35985 load.go:115] Finished reloading
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0813 02:18:07.422631   35985 collector.go:107] schema extensions wrote into \"/home/user/studio/dataconnect/.dataconnect/schema\"\nGenerated sources: main/input.gql [54370B] main/relation.gql [30196B] main/query.gql [7747B] main/mutation.gql [21010B] prelude.gql [73172B] main/implicit.gql [3592B] \nI0813 02:18:07.422667   35985 load.go:115] Finished reloading\n"}}
[debug] [2025-08-13T02:19:33.988Z] E0813 02:19:33.929329   35985 load.go:23] Could not load config from "/home/user/studio/dataconnect": could not unmarshal dataconnect.yaml at `/home/user/studio/dataconnect/dataconnect.yaml`: yaml: unmarshal errors:
  line 13: mapping key "specVersion" already defined at line 1
  line 14: mapping key "serviceId" already defined at line 2
  line 15: mapping key "location" already defined at line 3
  line 16: mapping key "schema" already defined at line 4
  line 24: mapping key "connectorDirs" already defined at line 12
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0813 02:19:33.929329   35985 load.go:23] Could not load config from \"/home/user/studio/dataconnect\": could not unmarshal dataconnect.yaml at `/home/user/studio/dataconnect/dataconnect.yaml`: yaml: unmarshal errors:\n  line 13: mapping key \"specVersion\" already defined at line 1\n  line 14: mapping key \"serviceId\" already defined at line 2\n  line 15: mapping key \"location\" already defined at line 3\n  line 16: mapping key \"schema\" already defined at line 4\n  line 24: mapping key \"connectorDirs\" already defined at line 12\n"}}
[debug] [2025-08-13T02:19:37.376Z] E0813 02:19:37.376240   35985 load.go:23] Could not load config from "/home/user/studio/dataconnect": could not unmarshal dataconnect.yaml at `/home/user/studio/dataconnect/dataconnect.yaml`: yaml: unmarshal errors:
  line 13: mapping key "specVersion" already defined at line 1
  line 14: mapping key "serviceId" already defined at line 2
  line 15: mapping key "location" already defined at line 3
  line 16: mapping key "schema" already defined at line 4
  line 24: mapping key "connectorDirs" already defined at line 12
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0813 02:19:37.376240   35985 load.go:23] Could not load config from \"/home/user/studio/dataconnect\": could not unmarshal dataconnect.yaml at `/home/user/studio/dataconnect/dataconnect.yaml`: yaml: unmarshal errors:\n  line 13: mapping key \"specVersion\" already defined at line 1\n  line 14: mapping key \"serviceId\" already defined at line 2\n  line 15: mapping key \"location\" already defined at line 3\n  line 16: mapping key \"schema\" already defined at line 4\n  line 24: mapping key \"connectorDirs\" already defined at line 12\n"}}
[debug] [2025-08-13T02:19:47.375Z] E0813 02:19:47.374817   35985 load.go:23] Could not load config from "/home/user/studio/dataconnect": could not unmarshal dataconnect.yaml at `/home/user/studio/dataconnect/dataconnect.yaml`: yaml: unmarshal errors:
  line 13: mapping key "specVersion" already defined at line 1
  line 14: mapping key "serviceId" already defined at line 2
  line 15: mapping key "location" already defined at line 3
  line 16: mapping key "schema" already defined at line 4
  line 24: mapping key "connectorDirs" already defined at line 12
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0813 02:19:47.374817   35985 load.go:23] Could not load config from \"/home/user/studio/dataconnect\": could not unmarshal dataconnect.yaml at `/home/user/studio/dataconnect/dataconnect.yaml`: yaml: unmarshal errors:\n  line 13: mapping key \"specVersion\" already defined at line 1\n  line 14: mapping key \"serviceId\" already defined at line 2\n  line 15: mapping key \"location\" already defined at line 3\n  line 16: mapping key \"schema\" already defined at line 4\n  line 24: mapping key \"connectorDirs\" already defined at line 12\n"}}
[debug] [2025-08-13T02:19:57.375Z] E0813 02:19:57.375622   35985 load.go:23] Could not load config from "/home/user/studio/dataconnect": could not unmarshal dataconnect.yaml at `/home/user/studio/dataconnect/dataconnect.yaml`: yaml: unmarshal errors:
  line 13: mapping key "specVersion" already defined at line 1
  line 14: mapping key "serviceId" already defined at line 2
  line 15: mapping key "location" already defined at line 3
  line 16: mapping key "schema" already defined at line 4
  line 24: mapping key "connectorDirs" already defined at line 12
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0813 02:19:57.375622   35985 load.go:23] Could not load config from \"/home/user/studio/dataconnect\": could not unmarshal dataconnect.yaml at `/home/user/studio/dataconnect/dataconnect.yaml`: yaml: unmarshal errors:\n  line 13: mapping key \"specVersion\" already defined at line 1\n  line 14: mapping key \"serviceId\" already defined at line 2\n  line 15: mapping key \"location\" already defined at line 3\n  line 16: mapping key \"schema\" already defined at line 4\n  line 24: mapping key \"connectorDirs\" already defined at line 12\n"}}
[debug] [2025-08-13T02:20:07.380Z] E0813 02:20:07.379872   35985 load.go:23] Could not load config from "/home/user/studio/dataconnect": could not unmarshal dataconnect.yaml at `/home/user/studio/dataconnect/dataconnect.yaml`: yaml: unmarshal errors:
  line 13: mapping key "specVersion" already defined at line 1
  line 14: mapping key "serviceId" already defined at line 2
  line 15: mapping key "location" already defined at line 3
  line 16: mapping key "schema" already defined at line 4
  line 24: mapping key "connectorDirs" already defined at line 12
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0813 02:20:07.379872   35985 load.go:23] Could not load config from \"/home/user/studio/dataconnect\": could not unmarshal dataconnect.yaml at `/home/user/studio/dataconnect/dataconnect.yaml`: yaml: unmarshal errors:\n  line 13: mapping key \"specVersion\" already defined at line 1\n  line 14: mapping key \"serviceId\" already defined at line 2\n  line 15: mapping key \"location\" already defined at line 3\n  line 16: mapping key \"schema\" already defined at line 4\n  line 24: mapping key \"connectorDirs\" already defined at line 12\n"}}
[debug] [2025-08-13T02:20:17.375Z] E0813 02:20:17.375435   35985 load.go:23] Could not load config from "/home/user/studio/dataconnect": could not unmarshal dataconnect.yaml at `/home/user/studio/dataconnect/dataconnect.yaml`: yaml: unmarshal errors:
  line 13: mapping key "specVersion" already defined at line 1
  line 14: mapping key "serviceId" already defined at line 2
  line 15: mapping key "location" already defined at line 3
  line 16: mapping key "schema" already defined at line 4
  line 24: mapping key "connectorDirs" already defined at line 12
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0813 02:20:17.375435   35985 load.go:23] Could not load config from \"/home/user/studio/dataconnect\": could not unmarshal dataconnect.yaml at `/home/user/studio/dataconnect/dataconnect.yaml`: yaml: unmarshal errors:\n  line 13: mapping key \"specVersion\" already defined at line 1\n  line 14: mapping key \"serviceId\" already defined at line 2\n  line 15: mapping key \"location\" already defined at line 3\n  line 16: mapping key \"schema\" already defined at line 4\n  line 24: mapping key \"connectorDirs\" already defined at line 12\n"}}
[debug] [2025-08-13T02:20:27.375Z] E0813 02:20:27.375588   35985 load.go:23] Could not load config from "/home/user/studio/dataconnect": could not unmarshal dataconnect.yaml at `/home/user/studio/dataconnect/dataconnect.yaml`: yaml: unmarshal errors:
  line 13: mapping key "specVersion" already defined at line 1
  line 14: mapping key "serviceId" already defined at line 2
  line 15: mapping key "location" already defined at line 3
  line 16: mapping key "schema" already defined at line 4
  line 24: mapping key "connectorDirs" already defined at line 12
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0813 02:20:27.375588   35985 load.go:23] Could not load config from \"/home/user/studio/dataconnect\": could not unmarshal dataconnect.yaml at `/home/user/studio/dataconnect/dataconnect.yaml`: yaml: unmarshal errors:\n  line 13: mapping key \"specVersion\" already defined at line 1\n  line 14: mapping key \"serviceId\" already defined at line 2\n  line 15: mapping key \"location\" already defined at line 3\n  line 16: mapping key \"schema\" already defined at line 4\n  line 24: mapping key \"connectorDirs\" already defined at line 12\n"}}
[debug] [2025-08-13T02:20:37.374Z] E0813 02:20:37.374623   35985 load.go:23] Could not load config from "/home/user/studio/dataconnect": could not unmarshal dataconnect.yaml at `/home/user/studio/dataconnect/dataconnect.yaml`: yaml: unmarshal errors:
  line 13: mapping key "specVersion" already defined at line 1
  line 14: mapping key "serviceId" already defined at line 2
  line 15: mapping key "location" already defined at line 3
  line 16: mapping key "schema" already defined at line 4
  line 24: mapping key "connectorDirs" already defined at line 12
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0813 02:20:37.374623   35985 load.go:23] Could not load config from \"/home/user/studio/dataconnect\": could not unmarshal dataconnect.yaml at `/home/user/studio/dataconnect/dataconnect.yaml`: yaml: unmarshal errors:\n  line 13: mapping key \"specVersion\" already defined at line 1\n  line 14: mapping key \"serviceId\" already defined at line 2\n  line 15: mapping key \"location\" already defined at line 3\n  line 16: mapping key \"schema\" already defined at line 4\n  line 24: mapping key \"connectorDirs\" already defined at line 12\n"}}
[debug] [2025-08-13T02:20:47.376Z] E0813 02:20:47.376355   35985 load.go:23] Could not load config from "/home/user/studio/dataconnect": could not unmarshal dataconnect.yaml at `/home/user/studio/dataconnect/dataconnect.yaml`: yaml: unmarshal errors:
  line 13: mapping key "specVersion" already defined at line 1
  line 14: mapping key "serviceId" already defined at line 2
  line 15: mapping key "location" already defined at line 3
  line 16: mapping key "schema" already defined at line 4
  line 24: mapping key "connectorDirs" already defined at line 12
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0813 02:20:47.376355   35985 load.go:23] Could not load config from \"/home/user/studio/dataconnect\": could not unmarshal dataconnect.yaml at `/home/user/studio/dataconnect/dataconnect.yaml`: yaml: unmarshal errors:\n  line 13: mapping key \"specVersion\" already defined at line 1\n  line 14: mapping key \"serviceId\" already defined at line 2\n  line 15: mapping key \"location\" already defined at line 3\n  line 16: mapping key \"schema\" already defined at line 4\n  line 24: mapping key \"connectorDirs\" already defined at line 12\n"}}
[debug] [2025-08-13T02:20:57.376Z] E0813 02:20:57.375712   35985 load.go:23] Could not load config from "/home/user/studio/dataconnect": could not unmarshal dataconnect.yaml at `/home/user/studio/dataconnect/dataconnect.yaml`: yaml: unmarshal errors:
  line 13: mapping key "specVersion" already defined at line 1
  line 14: mapping key "serviceId" already defined at line 2
  line 15: mapping key "location" already defined at line 3
  line 16: mapping key "schema" already defined at line 4
  line 24: mapping key "connectorDirs" already defined at line 12
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0813 02:20:57.375712   35985 load.go:23] Could not load config from \"/home/user/studio/dataconnect\": could not unmarshal dataconnect.yaml at `/home/user/studio/dataconnect/dataconnect.yaml`: yaml: unmarshal errors:\n  line 13: mapping key \"specVersion\" already defined at line 1\n  line 14: mapping key \"serviceId\" already defined at line 2\n  line 15: mapping key \"location\" already defined at line 3\n  line 16: mapping key \"schema\" already defined at line 4\n  line 24: mapping key \"connectorDirs\" already defined at line 12\n"}}
[debug] [2025-08-13T02:21:07.381Z] E0813 02:21:07.380882   35985 load.go:23] Could not load config from "/home/user/studio/dataconnect": could not unmarshal dataconnect.yaml at `/home/user/studio/dataconnect/dataconnect.yaml`: yaml: unmarshal errors:
  line 13: mapping key "specVersion" already defined at line 1
  line 14: mapping key "serviceId" already defined at line 2
  line 15: mapping key "location" already defined at line 3
  line 16: mapping key "schema" already defined at line 4
  line 24: mapping key "connectorDirs" already defined at line 12
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0813 02:21:07.380882   35985 load.go:23] Could not load config from \"/home/user/studio/dataconnect\": could not unmarshal dataconnect.yaml at `/home/user/studio/dataconnect/dataconnect.yaml`: yaml: unmarshal errors:\n  line 13: mapping key \"specVersion\" already defined at line 1\n  line 14: mapping key \"serviceId\" already defined at line 2\n  line 15: mapping key \"location\" already defined at line 3\n  line 16: mapping key \"schema\" already defined at line 4\n  line 24: mapping key \"connectorDirs\" already defined at line 12\n"}}
[debug] [2025-08-13T02:21:17.376Z] E0813 02:21:17.376217   35985 load.go:23] Could not load config from "/home/user/studio/dataconnect": could not unmarshal dataconnect.yaml at `/home/user/studio/dataconnect/dataconnect.yaml`: yaml: unmarshal errors:
  line 13: mapping key "specVersion" already defined at line 1
  line 14: mapping key "serviceId" already defined at line 2
  line 15: mapping key "location" already defined at line 3
  line 16: mapping key "schema" already defined at line 4
  line 24: mapping key "connectorDirs" already defined at line 12
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0813 02:21:17.376217   35985 load.go:23] Could not load config from \"/home/user/studio/dataconnect\": could not unmarshal dataconnect.yaml at `/home/user/studio/dataconnect/dataconnect.yaml`: yaml: unmarshal errors:\n  line 13: mapping key \"specVersion\" already defined at line 1\n  line 14: mapping key \"serviceId\" already defined at line 2\n  line 15: mapping key \"location\" already defined at line 3\n  line 16: mapping key \"schema\" already defined at line 4\n  line 24: mapping key \"connectorDirs\" already defined at line 12\n"}}
[debug] [2025-08-13T02:21:30.612Z] I0813 02:21:30.611958   35985 load.go:37] Reloading schema and connectors...
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0813 02:21:30.611958   35985 load.go:37] Reloading schema and connectors...\n"}}
[debug] [2025-08-13T02:21:30.697Z] I0813 02:21:30.696684   35985 control.go:84] [/emulator/studio 3c0e] UpdateResources(): done
Schema: sources: schema/schema.gql [928B] 
Connector "default": sources: 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0813 02:21:30.696684   35985 control.go:84] [/emulator/studio 3c0e] UpdateResources(): done\nSchema: sources: schema/schema.gql [928B] \nConnector \"default\": sources: \n"}}
[debug] [2025-08-13T02:21:30.772Z] E0813 02:21:30.772174   35985 load.go:101] Errors generating SDK: [connector "default"]: invalid Data Connect Connector no query found in connector
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0813 02:21:30.772174   35985 load.go:101] Errors generating SDK: [connector \"default\"]: invalid Data Connect Connector no query found in connector\n"}}
[debug] [2025-08-13T02:21:30.778Z] I0813 02:21:30.775286   35985 collector.go:107] schema extensions wrote into "/home/user/studio/dataconnect/.dataconnect/schema"
Generated sources: main/implicit.gql [2401B] prelude.gql [73172B] main/input.gql [47230B] main/relation.gql [24997B] main/query.gql [6570B] main/mutation.gql [17895B] 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0813 02:21:30.775286   35985 collector.go:107] schema extensions wrote into \"/home/user/studio/dataconnect/.dataconnect/schema\"\nGenerated sources: main/implicit.gql [2401B] prelude.gql [73172B] main/input.gql [47230B] main/relation.gql [24997B] main/query.gql [6570B] main/mutation.gql [17895B] \n"}}
[debug] [2025-08-13T02:21:30.780Z] I0813 02:21:30.779613   35985 load.go:115] Finished reloading
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0813 02:21:30.779613   35985 load.go:115] Finished reloading\n"}}
[debug] [2025-08-13T02:21:34.464Z] I0813 02:21:34.462639   35985 load.go:37] Reloading schema and connectors...
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0813 02:21:34.462639   35985 load.go:37] Reloading schema and connectors...\n"}}
[debug] [2025-08-13T02:21:34.649Z] I0813 02:21:34.647412   35985 collector.go:107] schema extensions wrote into "/home/user/studio/dataconnect/.dataconnect/schema"
Generated sources: main/query.gql [6570B] main/mutation.gql [17895B] main/implicit.gql [2401B] prelude.gql [73172B] main/input.gql [47230B] main/relation.gql [24997B] 
I0813 02:21:34.647489   35985 load.go:115] Finished reloading
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0813 02:21:34.647412   35985 collector.go:107] schema extensions wrote into \"/home/user/studio/dataconnect/.dataconnect/schema\"\nGenerated sources: main/query.gql [6570B] main/mutation.gql [17895B] main/implicit.gql [2401B] prelude.gql [73172B] main/input.gql [47230B] main/relation.gql [24997B] \nI0813 02:21:34.647489   35985 load.go:115] Finished reloading\n"}}
[debug] [2025-08-13T02:21:53.122Z] I0813 02:21:53.121395   35985 load.go:37] Reloading schema and connectors...
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0813 02:21:53.121395   35985 load.go:37] Reloading schema and connectors...\n"}}
[debug] [2025-08-13T02:21:53.185Z] E0813 02:21:53.183910   35985 load.go:101] Errors generating SDK: [connector "default"]: invalid Data Connect Connector no query found in connector
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0813 02:21:53.183910   35985 load.go:101] Errors generating SDK: [connector \"default\"]: invalid Data Connect Connector no query found in connector\n"}}
[debug] [2025-08-13T02:21:53.192Z] I0813 02:21:53.188331   35985 collector.go:107] schema extensions wrote into "/home/user/studio/dataconnect/.dataconnect/schema"
Generated sources: prelude.gql [73172B] main/implicit.gql [2401B] main/input.gql [47230B] main/relation.gql [24997B] main/query.gql [6570B] main/mutation.gql [17895B] 
I0813 02:21:53.188369   35985 load.go:115] Finished reloading
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0813 02:21:53.188331   35985 collector.go:107] schema extensions wrote into \"/home/user/studio/dataconnect/.dataconnect/schema\"\nGenerated sources: prelude.gql [73172B] main/implicit.gql [2401B] main/input.gql [47230B] main/relation.gql [24997B] main/query.gql [6570B] main/mutation.gql [17895B] \nI0813 02:21:53.188369   35985 load.go:115] Finished reloading\n"}}
[debug] [2025-08-14T07:31:47.029Z] [Firebase Plugin] Activating Firebase extension.
[info] [Firebase Plugin] Checked firebase-tools, is up to date!
[debug] [2025-08-14T07:32:24.233Z] [Firebase Plugin] requireAuthWrapper
[debug] [2025-08-14T07:32:24.241Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2025-08-14T07:32:24.273Z] [Firebase Plugin] VSCode notification server listening on port 40001
[info] [Firebase Plugin] Starting Data Connect toolkit (version 2.10.1) on port 50001
[info] i  dataconnect: downloading dataconnect-emulator-2.10.1... {"metadata":{"emulator":{"name":"dataconnect"},"message":"downloading dataconnect-emulator-2.10.1..."}}
[debug] [2025-08-14T07:32:27.057Z] >>> [apiv2][query] GET https://storage.googleapis.com/firemat-preview-drop/emulator/dataconnect-emulator-linux-v2.10.1 
[debug] [2025-08-14T07:32:27.359Z] <<< [apiv2][status] GET https://storage.googleapis.com/firemat-preview-drop/emulator/dataconnect-emulator-linux-v2.10.1 200
[debug] [2025-08-14T07:32:27.361Z] <<< [apiv2][body] GET https://storage.googleapis.com/firemat-preview-drop/emulator/dataconnect-emulator-linux-v2.10.1 [stream]
[info] i  dataconnect: Removing outdated emulator files: dataconnect-emulator-2.11.0 {"metadata":{"emulator":{"name":"dataconnect"},"message":"Removing outdated emulator files: dataconnect-emulator-2.11.0"}}
[debug] [2025-08-14T07:32:37.824Z] I0814 07:32:37.747609    2140 control.go:84] [/emulator/studio df10] UpdateResources(): done
Schema: sources: schema/schema.gql [928B] 
Connector "default": sources: 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0814 07:32:37.747609    2140 control.go:84] [/emulator/studio df10] UpdateResources(): done\nSchema: sources: schema/schema.gql [928B] \nConnector \"default\": sources: \n"}}
[debug] [2025-08-14T07:32:37.826Z] Ignoring unsupported arg: auto_download {"metadata":{"emulator":{"name":"dataconnect"},"message":"Ignoring unsupported arg: auto_download"}}
[debug] [2025-08-14T07:32:37.827Z] Ignoring unsupported arg: port {"metadata":{"emulator":{"name":"dataconnect"},"message":"Ignoring unsupported arg: port"}}
[debug] [2025-08-14T07:32:37.828Z] Starting Data Connect Emulator with command {"binary":"/home/user/.cache/firebase/emulators/dataconnect-emulator-2.10.1","args":["--logtostderr","-v=2","dev","--listen=localhost:50001","--config_dir=/home/user/studio/dataconnect","--enable_output_schema_extensions=true","--enable_output_generated_sdk=true"],"optionalArgs":["listen","config_dir","enable_output_schema_extensions","enable_output_generated_sdk"],"joinArgs":true,"shell":false,"port":9399} {"metadata":{"emulator":{"name":"dataconnect"},"message":"Starting Data Connect Emulator with command {\"binary\":\"/home/user/.cache/firebase/emulators/dataconnect-emulator-2.10.1\",\"args\":[\"--logtostderr\",\"-v=2\",\"dev\",\"--listen=localhost:50001\",\"--config_dir=/home/user/studio/dataconnect\",\"--enable_output_schema_extensions=true\",\"--enable_output_generated_sdk=true\"],\"optionalArgs\":[\"listen\",\"config_dir\",\"enable_output_schema_extensions\",\"enable_output_generated_sdk\"],\"joinArgs\":true,\"shell\":false,\"port\":9399}"}}
[info] i  dataconnect: Data Connect Emulator logging to dataconnect-debug.log {"metadata":{"emulator":{"name":"dataconnect"},"message":"Data Connect Emulator logging to dataconnect-debug.log"}}
[debug] [2025-08-14T07:32:37.900Z] I0814 07:32:37.884947    2145 load.go:37] Reloading schema and connectors...
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0814 07:32:37.884947    2145 load.go:37] Reloading schema and connectors...\n"}}
[debug] [2025-08-14T07:32:38.079Z] I0814 07:32:38.078349    2145 control.go:84] [/emulator/studio 4af2] UpdateResources(): done
Schema: sources: schema/schema.gql [928B] 
Connector "default": sources: 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0814 07:32:38.078349    2145 control.go:84] [/emulator/studio 4af2] UpdateResources(): done\nSchema: sources: schema/schema.gql [928B] \nConnector \"default\": sources: \n"}}
[debug] [2025-08-14T07:32:38.259Z] E0814 07:32:38.259139    2145 load.go:101] Errors generating SDK: [connector "default"]: invalid Data Connect Connector no query found in connector
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0814 07:32:38.259139    2145 load.go:101] Errors generating SDK: [connector \"default\"]: invalid Data Connect Connector no query found in connector\n"}}
[debug] [2025-08-14T07:32:38.265Z] I0814 07:32:38.264315    2145 collector.go:107] schema extensions wrote into "/home/user/studio/dataconnect/.dataconnect/schema"
Generated sources: main/implicit.gql [2401B] main/input.gql [47230B] main/relation.gql [24997B] prelude.gql [73172B] main/query.gql [6570B] main/mutation.gql [17895B] 
I0814 07:32:38.264399    2145 load.go:115] Finished reloading
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0814 07:32:38.264315    2145 collector.go:107] schema extensions wrote into \"/home/user/studio/dataconnect/.dataconnect/schema\"\nGenerated sources: main/implicit.gql [2401B] main/input.gql [47230B] main/relation.gql [24997B] prelude.gql [73172B] main/query.gql [6570B] main/mutation.gql [17895B] \nI0814 07:32:38.264399    2145 load.go:115] Finished reloading\n"}}
[debug] [2025-08-14T07:32:38.290Z] I0814 07:32:38.278111    2145 dev.go:95] Listening on address (HTTP + gRPC): 127.0.0.1:50001
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0814 07:32:38.278111    2145 dev.go:95] Listening on address (HTTP + gRPC): 127.0.0.1:50001\n"}}
[error] [Firebase Plugin] error:  {}
[debug] [2025-08-14T07:32:48.808Z] Running auto auth
[debug] [2025-08-14T07:32:48.838Z] No OAuth tokens found
[debug] [2025-08-14T07:32:48.844Z] [Firebase Plugin] User found:  smartdisputecanada@gmail.com
[info] [Firebase Plugin] (Core:Project) New user detected, fetching projects
[debug] [2025-08-14T07:32:48.849Z] No OAuth tokens found
[debug] [2025-08-14T07:32:48.851Z] >>> [apiv2][query] GET https://firebase.googleapis.com/v1beta1/projects pageSize=1000
[debug] [2025-08-14T07:32:49.234Z] <<< [apiv2][status] GET https://firebase.googleapis.com/v1beta1/projects 200
[debug] [2025-08-14T07:32:49.234Z] <<< [apiv2][body] GET https://firebase.googleapis.com/v1beta1/projects [omitted]
[debug] [2025-08-14T17:36:34.630Z] [Firebase Plugin] Activating Firebase extension.
[info] [Firebase Plugin] Checked firebase-tools, is up to date!
[debug] [2025-08-14T17:36:42.613Z] [Firebase Plugin] requireAuthWrapper
[debug] [2025-08-14T17:36:42.615Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2025-08-14T17:36:42.636Z] [Firebase Plugin] VSCode notification server listening on port 40001
[info] [Firebase Plugin] Starting Data Connect toolkit (version 2.10.1) on port 50001
[debug] [2025-08-14T17:36:45.802Z] I0814 17:36:45.685779     531 control.go:84] [/emulator/studio 1c50] UpdateResources(): done
Schema: sources: schema/schema.gql [928B] 
Connector "default": sources: 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0814 17:36:45.685779     531 control.go:84] [/emulator/studio 1c50] UpdateResources(): done\nSchema: sources: schema/schema.gql [928B] \nConnector \"default\": sources: \n"}}
[debug] [2025-08-14T17:36:45.805Z] Ignoring unsupported arg: auto_download {"metadata":{"emulator":{"name":"dataconnect"},"message":"Ignoring unsupported arg: auto_download"}}
[debug] [2025-08-14T17:36:45.807Z] Ignoring unsupported arg: port {"metadata":{"emulator":{"name":"dataconnect"},"message":"Ignoring unsupported arg: port"}}
[debug] [2025-08-14T17:36:45.815Z] Starting Data Connect Emulator with command {"binary":"/home/user/.cache/firebase/emulators/dataconnect-emulator-2.10.1","args":["--logtostderr","-v=2","dev","--listen=localhost:50001","--config_dir=/home/user/studio/dataconnect","--enable_output_schema_extensions=true","--enable_output_generated_sdk=true"],"optionalArgs":["listen","config_dir","enable_output_schema_extensions","enable_output_generated_sdk"],"joinArgs":true,"shell":false,"port":9399} {"metadata":{"emulator":{"name":"dataconnect"},"message":"Starting Data Connect Emulator with command {\"binary\":\"/home/user/.cache/firebase/emulators/dataconnect-emulator-2.10.1\",\"args\":[\"--logtostderr\",\"-v=2\",\"dev\",\"--listen=localhost:50001\",\"--config_dir=/home/user/studio/dataconnect\",\"--enable_output_schema_extensions=true\",\"--enable_output_generated_sdk=true\"],\"optionalArgs\":[\"listen\",\"config_dir\",\"enable_output_schema_extensions\",\"enable_output_generated_sdk\"],\"joinArgs\":true,\"shell\":false,\"port\":9399}"}}
[info] i  dataconnect: Data Connect Emulator logging to dataconnect-debug.log {"metadata":{"emulator":{"name":"dataconnect"},"message":"Data Connect Emulator logging to dataconnect-debug.log"}}
[debug] [2025-08-14T17:36:46.035Z] I0814 17:36:45.919172     536 load.go:37] Reloading schema and connectors...
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0814 17:36:45.919172     536 load.go:37] Reloading schema and connectors...\n"}}
[debug] [2025-08-14T17:36:46.483Z] I0814 17:36:46.301585     536 control.go:84] [/emulator/studio 7160] UpdateResources(): done
Schema: sources: schema/schema.gql [928B] 
Connector "default": sources: 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0814 17:36:46.301585     536 control.go:84] [/emulator/studio 7160] UpdateResources(): done\nSchema: sources: schema/schema.gql [928B] \nConnector \"default\": sources: \n"}}
[debug] [2025-08-14T17:36:49.950Z] E0814 17:36:46.584006     536 load.go:101] Errors generating SDK: [connector "default"]: invalid Data Connect Connector no query found in connector
I0814 17:36:46.590915     536 collector.go:107] schema extensions wrote into "/home/user/studio/dataconnect/.dataconnect/schema"
Generated sources: prelude.gql [73172B] main/relation.gql [24997B] main/query.gql [6570B] main/mutation.gql [17895B] main/implicit.gql [2401B] main/input.gql [47230B] 
I0814 17:36:46.593084     536 load.go:115] Finished reloading
I0814 17:36:46.609165     536 dev.go:95] Listening on address (HTTP + gRPC): 127.0.0.1:50001
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0814 17:36:46.584006     536 load.go:101] Errors generating SDK: [connector \"default\"]: invalid Data Connect Connector no query found in connector\nI0814 17:36:46.590915     536 collector.go:107] schema extensions wrote into \"/home/user/studio/dataconnect/.dataconnect/schema\"\nGenerated sources: prelude.gql [73172B] main/relation.gql [24997B] main/query.gql [6570B] main/mutation.gql [17895B] main/implicit.gql [2401B] main/input.gql [47230B] \nI0814 17:36:46.593084     536 load.go:115] Finished reloading\nI0814 17:36:46.609165     536 dev.go:95] Listening on address (HTTP + gRPC): 127.0.0.1:50001\n"}}
[error] [Firebase Plugin] error:  {}
[debug] [2025-08-14T17:37:57.863Z] Running auto auth
[debug] [2025-08-14T17:37:57.898Z] No OAuth tokens found
[debug] [2025-08-14T17:37:57.899Z] [Firebase Plugin] User found:  smartdisputecanada@gmail.com
[info] [Firebase Plugin] (Core:Project) New user detected, fetching projects
[debug] [2025-08-14T17:37:57.905Z] No OAuth tokens found
[debug] [2025-08-14T17:37:57.911Z] >>> [apiv2][query] GET https://firebase.googleapis.com/v1beta1/projects pageSize=1000
[debug] [2025-08-14T17:37:58.430Z] <<< [apiv2][status] GET https://firebase.googleapis.com/v1beta1/projects 200
[debug] [2025-08-14T17:37:58.431Z] <<< [apiv2][body] GET https://firebase.googleapis.com/v1beta1/projects [omitted]
[debug] [2025-08-14T17:42:52.127Z] [Firebase Plugin] Value of process.env.MONOSPACE_ENV: true
[debug] [2025-08-14T17:42:52.128Z] [Firebase Plugin] requireAuthWrapper
[debug] [2025-08-14T17:42:52.130Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2025-08-14T17:42:52.130Z] > authorizing via signed-in user (smartdisputecanada@gmail.com)
[debug] [2025-08-14T17:42:52.131Z] No OAuth tokens found
[debug] [2025-08-14T17:42:52.131Z] [Firebase Plugin] User found:  smartdisputecanada@gmail.com
[info] [Firebase Plugin] (Core:Project) New user detected, fetching projects
[debug] [2025-08-14T17:42:52.133Z] No OAuth tokens found
[debug] [2025-08-14T17:42:52.136Z] >>> [apiv2][query] GET https://firebase.googleapis.com/v1beta1/projects pageSize=1000
[debug] [2025-08-14T17:42:52.178Z] [Firebase Plugin] Value of process.env.MONOSPACE_ENV: true
[debug] [2025-08-14T17:42:52.179Z] [Firebase Plugin] requireAuthWrapper
[debug] [2025-08-14T17:42:52.180Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2025-08-14T17:42:52.180Z] > authorizing via signed-in user (smartdisputecanada@gmail.com)
[debug] [2025-08-14T17:42:52.183Z] No OAuth tokens found
[debug] [2025-08-14T17:42:52.184Z] [Firebase Plugin] User found:  smartdisputecanada@gmail.com
[info] [Firebase Plugin] (Core:Project) New user detected, fetching projects
[debug] [2025-08-14T17:42:52.185Z] No OAuth tokens found
[debug] [2025-08-14T17:42:52.186Z] >>> [apiv2][query] GET https://firebase.googleapis.com/v1beta1/projects pageSize=1000
[debug] [2025-08-14T17:42:52.578Z] <<< [apiv2][status] GET https://firebase.googleapis.com/v1beta1/projects 200
[debug] [2025-08-14T17:42:52.578Z] <<< [apiv2][body] GET https://firebase.googleapis.com/v1beta1/projects [omitted]
[debug] [2025-08-14T17:42:52.585Z] <<< [apiv2][status] GET https://firebase.googleapis.com/v1beta1/projects 200
[debug] [2025-08-14T17:42:52.586Z] <<< [apiv2][body] GET https://firebase.googleapis.com/v1beta1/projects [omitted]
[debug] [2025-08-14T17:44:11.557Z] [Firebase Plugin] Activating Firebase extension.
[info] [Firebase Plugin] Checked firebase-tools, is up to date!
[debug] [2025-08-14T17:44:17.340Z] [Firebase Plugin] requireAuthWrapper
[debug] [2025-08-14T17:44:17.342Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2025-08-14T17:44:17.343Z] > authorizing via signed-in user (smartdisputecanada@gmail.com)
[debug] [2025-08-14T17:44:17.349Z] No OAuth tokens found
[debug] [2025-08-14T17:44:17.349Z] No OAuth tokens found
[debug] [2025-08-14T17:44:17.350Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2025-08-14T17:44:17.351Z] > authorizing via signed-in user (smartdisputecanada@gmail.com)
[debug] [2025-08-14T17:44:17.352Z] [Firebase Plugin] User found:  smartdisputecanada@gmail.com
[info] [Firebase Plugin] (Core:Project) New user detected, fetching projects
[debug] [2025-08-14T17:44:17.353Z] No OAuth tokens found
[debug] [2025-08-14T17:44:17.355Z] >>> [apiv2][query] GET https://firebase.googleapis.com/v1beta1/projects pageSize=1000
[debug] [2025-08-14T17:44:17.411Z] [Firebase Plugin] VSCode notification server listening on port 40002
[info] [Firebase Plugin] Starting Data Connect toolkit (version 2.10.1) on port 50002
[debug] [2025-08-14T17:44:20.165Z] I0814 17:44:20.127051    2684 control.go:84] [/emulator/studio f0ea] UpdateResources(): done
Schema: sources: schema/schema.gql [928B] 
Connector "default": sources: 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0814 17:44:20.127051    2684 control.go:84] [/emulator/studio f0ea] UpdateResources(): done\nSchema: sources: schema/schema.gql [928B] \nConnector \"default\": sources: \n"}}
[debug] [2025-08-14T17:44:20.167Z] Ignoring unsupported arg: auto_download {"metadata":{"emulator":{"name":"dataconnect"},"message":"Ignoring unsupported arg: auto_download"}}
[debug] [2025-08-14T17:44:20.167Z] Ignoring unsupported arg: port {"metadata":{"emulator":{"name":"dataconnect"},"message":"Ignoring unsupported arg: port"}}
[debug] [2025-08-14T17:44:20.168Z] Starting Data Connect Emulator with command {"binary":"/home/user/.cache/firebase/emulators/dataconnect-emulator-2.10.1","args":["--logtostderr","-v=2","dev","--listen=localhost:50002","--config_dir=/home/user/studio/dataconnect","--enable_output_schema_extensions=true","--enable_output_generated_sdk=true"],"optionalArgs":["listen","config_dir","enable_output_schema_extensions","enable_output_generated_sdk"],"joinArgs":true,"shell":false,"port":9399} {"metadata":{"emulator":{"name":"dataconnect"},"message":"Starting Data Connect Emulator with command {\"binary\":\"/home/user/.cache/firebase/emulators/dataconnect-emulator-2.10.1\",\"args\":[\"--logtostderr\",\"-v=2\",\"dev\",\"--listen=localhost:50002\",\"--config_dir=/home/user/studio/dataconnect\",\"--enable_output_schema_extensions=true\",\"--enable_output_generated_sdk=true\"],\"optionalArgs\":[\"listen\",\"config_dir\",\"enable_output_schema_extensions\",\"enable_output_generated_sdk\"],\"joinArgs\":true,\"shell\":false,\"port\":9399}"}}
[info] i  dataconnect: Data Connect Emulator logging to dataconnect-debug.log {"metadata":{"emulator":{"name":"dataconnect"},"message":"Data Connect Emulator logging to dataconnect-debug.log"}}
[debug] [2025-08-14T17:44:20.239Z] I0814 17:44:20.230578    2689 load.go:37] Reloading schema and connectors...
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0814 17:44:20.230578    2689 load.go:37] Reloading schema and connectors...\n"}}
[debug] [2025-08-14T17:44:20.271Z] <<< [apiv2][status] GET https://firebase.googleapis.com/v1beta1/projects 200
[debug] [2025-08-14T17:44:20.272Z] <<< [apiv2][body] GET https://firebase.googleapis.com/v1beta1/projects [omitted]
[debug] [2025-08-14T17:44:20.400Z] I0814 17:44:20.388126    2689 control.go:84] [/emulator/studio 5d67] UpdateResources(): done
Schema: sources: schema/schema.gql [928B] 
Connector "default": sources: 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0814 17:44:20.388126    2689 control.go:84] [/emulator/studio 5d67] UpdateResources(): done\nSchema: sources: schema/schema.gql [928B] \nConnector \"default\": sources: \n"}}
[debug] [2025-08-14T17:44:20.518Z] E0814 17:44:20.518130    2689 load.go:101] Errors generating SDK: [connector "default"]: invalid Data Connect Connector no query found in connector
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0814 17:44:20.518130    2689 load.go:101] Errors generating SDK: [connector \"default\"]: invalid Data Connect Connector no query found in connector\n"}}
[debug] [2025-08-14T17:44:20.526Z] I0814 17:44:20.521475    2689 collector.go:107] schema extensions wrote into "/home/user/studio/dataconnect/.dataconnect/schema"
Generated sources: main/implicit.gql [2401B] main/input.gql [47230B] main/relation.gql [24997B] main/query.gql [6570B] main/mutation.gql [17895B] prelude.gql [73172B] 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0814 17:44:20.521475    2689 collector.go:107] schema extensions wrote into \"/home/user/studio/dataconnect/.dataconnect/schema\"\nGenerated sources: main/implicit.gql [2401B] main/input.gql [47230B] main/relation.gql [24997B] main/query.gql [6570B] main/mutation.gql [17895B] prelude.gql [73172B] \n"}}
[debug] [2025-08-14T17:44:20.529Z] I0814 17:44:20.525390    2689 load.go:115] Finished reloading
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0814 17:44:20.525390    2689 load.go:115] Finished reloading\n"}}
[debug] [2025-08-14T17:44:20.569Z] I0814 17:44:20.544258    2689 dev.go:95] Listening on address (HTTP + gRPC): 127.0.0.1:50002
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0814 17:44:20.544258    2689 dev.go:95] Listening on address (HTTP + gRPC): 127.0.0.1:50002\n"}}
[error] [Firebase Plugin] error:  {}
[debug] [2025-08-14T21:08:52.333Z] [Firebase Plugin] Activating Firebase extension.
[info] [Firebase Plugin] Checked firebase-tools, is up to date!
[debug] [2025-08-14T21:09:04.550Z] [Firebase Plugin] requireAuthWrapper
[debug] [2025-08-14T21:09:04.555Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2025-08-14T21:09:04.588Z] [Firebase Plugin] VSCode notification server listening on port 40001
[error] [Firebase Plugin] requireAuth error: Could not refresh access token: Unsuccessful response status code. Request failed with status code 500
[debug] [2025-08-14T21:09:27.102Z] [Firebase Plugin] Activating Firebase extension.
[info] [Firebase Plugin] Checked firebase-tools, is up to date!
[debug] [2025-08-14T21:09:43.510Z] [Firebase Plugin] requireAuthWrapper
[debug] [2025-08-14T21:09:43.516Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2025-08-14T21:09:43.645Z] [Firebase Plugin] VSCode notification server listening on port 40002
[info] [Firebase Plugin] Starting Data Connect toolkit (version 2.10.1) on port 50001
[debug] [2025-08-14T21:09:45.426Z] I0814 21:09:45.392331     746 control.go:84] [/emulator/studio 98a1] UpdateResources(): done
Schema: sources: schema/schema.gql [928B] 
Connector "default": sources: 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0814 21:09:45.392331     746 control.go:84] [/emulator/studio 98a1] UpdateResources(): done\nSchema: sources: schema/schema.gql [928B] \nConnector \"default\": sources: \n"}}
[debug] [2025-08-14T21:09:45.429Z] Ignoring unsupported arg: auto_download {"metadata":{"emulator":{"name":"dataconnect"},"message":"Ignoring unsupported arg: auto_download"}}
[debug] [2025-08-14T21:09:45.431Z] Ignoring unsupported arg: port {"metadata":{"emulator":{"name":"dataconnect"},"message":"Ignoring unsupported arg: port"}}
[debug] [2025-08-14T21:09:45.432Z] Starting Data Connect Emulator with command {"binary":"/home/user/.cache/firebase/emulators/dataconnect-emulator-2.10.1","args":["--logtostderr","-v=2","dev","--listen=localhost:50001","--config_dir=/home/user/studio/dataconnect","--enable_output_schema_extensions=true","--enable_output_generated_sdk=true"],"optionalArgs":["listen","config_dir","enable_output_schema_extensions","enable_output_generated_sdk"],"joinArgs":true,"shell":false,"port":9399} {"metadata":{"emulator":{"name":"dataconnect"},"message":"Starting Data Connect Emulator with command {\"binary\":\"/home/user/.cache/firebase/emulators/dataconnect-emulator-2.10.1\",\"args\":[\"--logtostderr\",\"-v=2\",\"dev\",\"--listen=localhost:50001\",\"--config_dir=/home/user/studio/dataconnect\",\"--enable_output_schema_extensions=true\",\"--enable_output_generated_sdk=true\"],\"optionalArgs\":[\"listen\",\"config_dir\",\"enable_output_schema_extensions\",\"enable_output_generated_sdk\"],\"joinArgs\":true,\"shell\":false,\"port\":9399}"}}
[info] i  dataconnect: Data Connect Emulator logging to dataconnect-debug.log {"metadata":{"emulator":{"name":"dataconnect"},"message":"Data Connect Emulator logging to dataconnect-debug.log"}}
[debug] [2025-08-14T21:09:45.540Z] I0814 21:09:45.481996     751 load.go:37] Reloading schema and connectors...
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0814 21:09:45.481996     751 load.go:37] Reloading schema and connectors...\n"}}
[debug] [2025-08-14T21:09:46.103Z] I0814 21:09:45.629691     751 control.go:84] [/emulator/studio b37b] UpdateResources(): done
Schema: sources: schema/schema.gql [928B] 
Connector "default": sources: 
E0814 21:09:45.847528     751 load.go:101] Errors generating SDK: [connector "default"]: invalid Data Connect Connector no query found in connector
I0814 21:09:45.853894     751 collector.go:107] schema extensions wrote into "/home/user/studio/dataconnect/.dataconnect/schema"
Generated sources: main/relation.gql [24997B] main/query.gql [6570B] main/mutation.gql [17895B] prelude.gql [73172B] main/implicit.gql [2401B] main/input.gql [47230B] 
I0814 21:09:45.853957     751 load.go:115] Finished reloading
I0814 21:09:45.881123     751 dev.go:95] Listening on address (HTTP + gRPC): 127.0.0.1:50001
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0814 21:09:45.629691     751 control.go:84] [/emulator/studio b37b] UpdateResources(): done\nSchema: sources: schema/schema.gql [928B] \nConnector \"default\": sources: \nE0814 21:09:45.847528     751 load.go:101] Errors generating SDK: [connector \"default\"]: invalid Data Connect Connector no query found in connector\nI0814 21:09:45.853894     751 collector.go:107] schema extensions wrote into \"/home/user/studio/dataconnect/.dataconnect/schema\"\nGenerated sources: main/relation.gql [24997B] main/query.gql [6570B] main/mutation.gql [17895B] prelude.gql [73172B] main/implicit.gql [2401B] main/input.gql [47230B] \nI0814 21:09:45.853957     751 load.go:115] Finished reloading\nI0814 21:09:45.881123     751 dev.go:95] Listening on address (HTTP + gRPC): 127.0.0.1:50001\n"}}
[error] [Firebase Plugin] error:  {}
[debug] [2025-08-14T21:10:23.691Z] Running auto auth
[debug] [2025-08-14T21:10:23.712Z] No OAuth tokens found
[debug] [2025-08-14T21:10:23.713Z] [Firebase Plugin] User found:  smartdisputecanada@gmail.com
[info] [Firebase Plugin] (Core:Project) New user detected, fetching projects
[debug] [2025-08-14T21:10:23.718Z] No OAuth tokens found
[debug] [2025-08-14T21:10:23.725Z] >>> [apiv2][query] GET https://firebase.googleapis.com/v1beta1/projects pageSize=1000
[debug] [2025-08-14T21:10:24.029Z] <<< [apiv2][status] GET https://firebase.googleapis.com/v1beta1/projects 200
[debug] [2025-08-14T21:10:24.030Z] <<< [apiv2][body] GET https://firebase.googleapis.com/v1beta1/projects [omitted]
